Solutions/Corvus.Json.Specs/Features/JsonModel/JsonSerializationWithSerializer.feature

```diff
@@ -0,0 +1,400 @@
+	When the json value is round-tripped via Serialization enabling inefficient serialization
+	Then the round-tripped result should be <type>
+	And the round-tripped result should be equal to the JsonAny <jsonValue>
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+Scenario Outline: Serialize a dotnet-backed JsonAny to a string
+	Given the dotnet backed JsonAny <jsonValue>
+	When the json value is round-tripped via Serialization enabling inefficient serialization
+	Then the round-tripped result should be <type>
+	And the round-tripped result should be equal to the JsonAny <jsonValue>
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+
+Scenario Outline: Write a jsonelement-backed JsonAny to a string
+	Given the JsonElement backed JsonAny <jsonValue>
+	When the json value is round-tripped via Serialization enabling inefficient serialization
+	Then the round-tripped result should be <type>
+	And the round-tripped result should be equal to the JsonAny <jsonValue>
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+Scenario Outline: Write a dotnet-backed JsonAny to a string
+	Given the dotnet backed JsonAny <jsonValue>
+	When the json value is round-tripped via Serialization enabling inefficient serialization
+	Then the round-tripped result should be <type>
+	And the round-tripped result should be equal to the JsonAny <jsonValue>
+
+	Examples:
+		| jsonValue                               | type      |
+		| {"foo": 3, "bar": "hello", "baz": null} | an Object |
+		| [1,2,"3",4.0]                           | an Array  |
+		| true                                    | a Boolean |
+		| "Hello world"                           | a String  |
+		| 3.2                                     | a Number  |
+		| null                                    | Null      |
+
+Scenario Outline: Write a jsonelement-backed JsonNotAny to a string
```

Comment:
All the scenarios that follow look like they could have been reduced to a single `Scenario Outline` with a table in the `Examples` with a row for each variation. Was that what you had in mind when you wrote this (apparently empty) `Scenario Outline`?

Reply:
That seemed to be a hangover duplicated from the other serialization specs project. I've cleaned them both out.